frontend/src/components/feed/Feed.js

```diff
@@ -30,8 +34,10 @@ const Feed = ({ navigate }) => {
     if(token) {
       return(
         <>
-          <h2>Posts</h2>
-            <button onClick={logout}>
+          <h2 className='h2'>Posts</h2>
+            <button className='btn' onClick={() => setForm(!form) }>Add Post</button>
```

Comment:
We still have the same class name of 'btn' which I think is going to be problematic in future - can we rename it so that we have a better understanding of what button it is?